File: assg2/q2.py
#to define two matrices and find their sum and dot product
a = [2, 5, 7]
b = [6, 6, 8]
sum = []
dot = 0
for x in range(3):
    sum.append( a[x]+b[x] )
for x in range(3):
    dot= dot+ (a[x]*b[x] )
print("Sum of the matrices")
for x in sum:
    print(x)
print("Dot prodduct of the Matrices")
print(dot)

File: assg2/q2.c
/* Finding sum and dot product of two matrices */
#include <stdio.h>

void main(){
    int a[3] = { 7, 15, 3}, b[3] = {3, 10, 10};
    int sum[3], dot=0;
    for(int i = 0; i<3; i++){
        sum[i] = a[i] + b[i];
        dot = dot + (a[i]*b[i]);
    }
    printf("Sum of the Matrices: \n");
    for(int i = 0; i <3; i++){
        printf("%d\n", sum[i]);
    }
    printf("Dot product of Matrices = %d", dot);
}
